scoresheets/ObjectConfiguration.tex

```diff
@@ -2,23 +2,23 @@
 
 \begin{scorelist}
 	\scoreheading{Describing the objects} % 100
-	%\scoreitem[n]{50 \/ n}{Description touches upon each of the n objects}
-	\scoreitem[10]{5}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description touches upon each of the 10 objects}
 	\scoreitem[10]{5}{Description of each object is accurate}
 
 	\scoreheading{Move to Part 2} % 30
 	\scoreitem{30}{Follow official to the human room}
 
 	\scoreheading{Correct Human Error} % 130
 	\scoreitem{10}{Focus on the correct space}
-	\scoreitem[4]{13}{Identifies each of the p errors}
+	\scoreitem[4]{13}{Identifies each of the errors}
```

Comment:
Remove each, you are scoring once per error, so it should be something like _Clearly state the nature of an error_. Plus, I feel 4x13 leading to a weird number.